### ISSUE TITLE
Requesting for Smoke Test Help

### ISSUE THREAD

**Issue by chin-herng** [nus-cs2103-AY2324S2/forum#452]
Hi, I am looking for people to help with smoke testing my application. The release can be downloaded here: https://github.com/chin-herng/ip/releases/tag/v0.3

To clarify, as mentioned in the [user guide](https://chin-herng.github.io/ip/#exiting-the-program), the command to exit the program is `exit` rather than the default `bye` as given in the project template. This is an attempt to customise the chatbot, but also I find that `exit` would be more intuitive for a CLI application because most CLI applications use `exit` for termination instead of `bye`.

Thank you.

**Comment by yyccbb** [comment]
Testing on Windows 11.

Your app runs normally but it keeps asking "what" when I tried to use "bye" to exit it. Seems like "what" is the response for unrecognized command but it is not very intuitive.

**Comment by justinlengch** [comment]
Testing on windows 11.

When adding more than 1 "|" character on the bob.txt save file, your jar will not run. Perhaps implement better error handling in case save file is unintentionally corrupted so the program will not crash.

**Comment by chin-herng** [comment]
@yyccbb Thanks for the input! Part of the personality of the chatbot is to give informal and short replies similar to the text messages we send everyday.

**Comment by faaheem13** [comment]
Testing on windows 11.

When trying to add tasks, even though it acknowledges and remarks for wrong syntax, the program doesn't go forward when I try to input in a valid format. I do not receive acknowledgement of task added or otherwise. Error handling seems to be done for the tasks, however, it seems that the task itself isn't added or the text file to store the data isn't created on my end.

As mentioned earlier by another person, "bye" does result in a "what" response as well.

**Comment by chin-herng** [comment]
@justinlengch Thanks for pointing this out! Can you elaborate more on where exactly is the "|" character added? Perhaps it would be helpful to have a sample problematic data file so that debugging can be more convenient.

**Comment by chin-herng** [comment]
@faaheem13 Thanks for the input! Can you elaborate on what does it mean by "the program doesn't go forward"? Did the program terminate, or did you not receive any reply from the chatbot upon entering your input (i.e. only one dialog box with your input)? Are you able to check the directory containing the jar file and see whether there exists a directory named "data" with the file "bob.txt"?

Additionally, I have edited the issue description to clarify about the command used to exit the program.

**Comment by faaheem13** [comment]
@chin-herng By "the program doesn't go forward", I meant that I did not receive any reply from the chatbot upon entering my input (the Send button does not seem to register the input). The directory containing the jar file does not have a bob.txt file, in which case I would recommend handling the case of "if .txt file not found, create a .txt file" and proceed there on.

However, it could also be a unique case on my laptop, so I would ask you to clarify with the rest who have put it through the smoke test as well.

Hope that helps!

**Comment by chin-herng** [comment]
@faaheem13 Thanks for the reply. Unfortunately, the issues could not be reproduced from my end. I tried entering a command in a wrong syntax followed by a valid one. I did receive an acknowledgement that the task has been added. I have attached a screenshot below for clarity.

![image](https://github.com/nus-cs2103-AY2324S2/forum/assets/44641849/c4eb92ee-32af-4729-af58-b4df7c11fae4)

I ran the jar file in a new folder containing only the jar file. The directory as well as the data file was automatically created as expected. I tested my program on Windows 11 as well. In this case, it might be helpful to give more detailed Windows specifications. I am using the Windows 11 Home Single Language edition with version 22H2.
